### DISCUSSION TITLE
How to Integrate Google login when using sycamore?

### DISCUSSION THREAD

**Discussion by zwl1619** [sycamore-rs/sycamore#350] [Q&A]
How to Integrate Google login when using sycamore?

**Comment by lukechu10** [comment]
There isn't an integrated way for now so I believe your best bet is to use a combination of `web-sys` + `wasm-bindgen` to directly use the JS apis. See https://rustwasm.github.io/docs/wasm-bindgen/reference/js-snippets.html for more information
